### PR TITLE
User Register/Login UI

### LINKED ISSUE #14: Connect User Register/Login to the proper UI

### PR DESCRIPTION
**Pull request by thanujasiva**
Example process:

Initial page:
image

Registering an account:
image
image

Logging into an account:
image
image
image

Home page:
image

### CODE REVIEW

File: src/main/resources/templates/register-login.html [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
-    <link rel="stylesheet" href="/src/main/resources/static/register-login.css">
+    <link rel="stylesheet" th:href="@{../css/register-login.css}" />
 </head>
 <body>
     <div class="header">
         <h1>Online Book Store</h1>
     </div>
     <div class="container-register-login">
-        <a href="register.html">
+        <a href="http://localhost:8080/register">
```
Comment (waheebhashmi) [inline src/main/resources/templates/register-login.html]: would this work on the azure site?

Comment (waheebhashmi) [reply src/main/resources/templates/register-login.html]: same in createAccountResult and accountError

Comment (thanujasiva) [reply src/main/resources/templates/register-login.html]: Thanks! Just fixed the hrefs

**Review by maishabd23 — APPROVED** [review]

**Review by shrimeichock — APPROVED** [review]

File: src/main/resources/templates/register.html [hunk 2]
```diff
@@ -13,18 +13,19 @@ <h1>Online Book Store</h1>
         <div>
             <h1>Register</h1>
         </div>
-        <form action="/register" method="post">
+        <form action="/register" th:action="@{/register}" th:object="${user}" method="post">
```
Comment (shrimeichock) [inline src/main/resources/templates/register.html]: multiple 'action' attributes

File: src/main/resources/templates/register.html [hunk 3]
```diff
@@ -13,18 +13,19 @@ <h1>Online Book Store</h1>
...
-                <input type="email" id="email" name="email" required>
+                <label for="password">Password:</label><br>
+                <input type="password" id="password" name="password" th:field="*{password}" required>
             </div>
             <div class="input-div">
-                <label for="password">Password:</label><br>
-                <input type="password" id="password" name="password" required>
+                <label for="userType1">User Type</label><br>
+                <input id="userType1" name="userType" type="radio" value="BOOKUSER" checked="checked" th:field="*{userType}"/><label for="userType1">Bookstore User</label>
+                <input id="userType2" name="userType" type="radio" value="BOOKOWNER"/><label for="userType1" th:field="*{userType}">Bookstore Owner</label>
```
Comment (shrimeichock) [inline src/main/resources/templates/register.html]: label should be for 'userType2'

**Review by shrimeichock — CHANGES_REQUESTED** [review]

**Review by waheebhashmi — APPROVED** [review]

**Review by shrimeichock — APPROVED** [review]

**Pull request merged**